crates/oxc_minifier/src/ast_passes/replace_global_defines.rs

```diff
@@ -84,8 +113,8 @@ impl<'a> ReplaceGlobalDefines<'a> {
     }
 
     fn replace_identifier_defines(&self, expr: &mut Expression<'a>) {
-        for (key, value) in &self.config.0.identifier_defines {
-            if let Expression::Identifier(ident) = expr {
+        if let Expression::Identifier(ident) = expr {
+            for (key, value) in &self.config.0.identifier_defines {
```

Comment:
i changed this to avoid looping over `identifier_defines` if we are not processing an identifier